conf/scheduled_document_upload.routes

```diff
@@ -0,0 +1 @@
+GET         /scheduled/scheduled_document_upload/upload                 uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.schedule.ScheduledDocumentController.uploadScheduledDocument()
```

Comment:
since we are going to move all the file upload controllers into one package, it now makes sense to re-organise the routes so create one routes file called `file_upload.routes" and put the routes for the supporting-evidence and schedule-mrn upload in there

Reply:
`/scheduled/scheduled_document_upload/upload` - do not use underscore, use a hyphen.

I know the ticket had underscore but it was raised in refinement last week that we should not do that.